ccm_web/client/src/components/CreateSelectSectionWidget.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { makeStyles, Tooltip, Typography } from '@material-ui/core'
+import HelpOutline from '@material-ui/icons/HelpOutline'
+
+import CreateSectionWidget from './CreateSectionWidget'
+import SectionSelectorWidget, { SelectableCanvasCourseSection } from './SectionSelectorWidget'
+import { CanvasCourseBase, CanvasCourseSection } from '../models/canvas'
+
+const useStyles = makeStyles((theme) => ({
+  createSectionWidget: {
+    width: '500px'
+  },
+  sectionSelectionContainer: {
+    textAlign: 'center',
+    maxHeight: '400px'
+  },
+  newSectionHint: {
+    display: 'flex'
+  },
+  tooltip: {
+    marginLeft: theme.spacing(1)
+  },
+  createSectionContainer: {
+    paddingBottom: theme.spacing(2)
+  }
+}))
+
+interface CreateSelectSectionWidgetProps {
+  course: CanvasCourseBase
+  sections: SelectableCanvasCourseSection[]
+  selectedSection?: SelectableCanvasCourseSection
+  setSelectedSection: (section: SelectableCanvasCourseSection) => void
+  onSectionCreated: (section: CanvasCourseSection) => void
+}
+
+export default function CreateSelectSectionWidget (props: CreateSelectSectionWidgetProps): JSX.Element {
+  const classes = useStyles()
+  return (
+    <>
+      <div className={classes.createSectionContainer}>
+        <div className={classes.newSectionHint}>
+          <Typography>Create a new section to add users</Typography>
+          <Tooltip className={classes.tooltip} placement='top' title='Enter a distinct name for this section'>
+            <HelpOutline fontSize='small'/>
+          </Tooltip>
```

Comment:
may be a small space between the text and textbox. I know the PR is still a draft. But just shared my observation. 
![Screen Shot 2021-12-17 at 4 32 42 PM](https://user-images.githubusercontent.com/8579775/146610525-d2c87324-b387-4cff-a4e7-f229a4e327bb.png)
?

Reply:
That's a component I'm not touching, so maybe this can be a separate issue. We should probably add validation to that field too.

Reply:
Agree though, FWIW

Reply:
I think I took care of this actually, it made sense to do it in the composite component you commented on.